src/app/view/components/login/main/dot-login-page.component.scss

```diff
@@ -5,6 +5,15 @@
     width: 100%;
     display: flex;
     justify-content: center;
+
+    ::ng-deep {
+        .error-message {
+            min-height: $line-height;
```

Comment:
Why here and no directly in the component?

Reply:
don't repeat the rule 3 times.. in each component.